sql/util_internal_table_ddl.sql

```diff
@@ -74,6 +74,15 @@ BEGIN
 END
 $BODY$;
 
+CREATE OR REPLACE FUNCTION _timescaledb_internal.closed_dimension_is_finite(
+    val      BIGINT
+)
+    RETURNS BOOLEAN LANGUAGE SQL IMMUTABLE AS
+$BODY$
+    SELECT val >= 0 AND val <= 2147483647
```

Comment:
So I decided not to make NULL since all the C code for conflict resolution etc needs actual values. I can convert the values to NULL on write and back to constants on read in C but this seemed somewhat silly. What do you think?